packages/@aws-cdk/aws-eks/test/sdk-call-integ-test-docker-app/app/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:16-alpine3.13
+
+# Create app directory
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+# Install app dependencies
+COPY package.json /usr/src/app/
+RUN yarn install
```

Comment:
I'd use `npm ci` here to install the package-json.lock, yarn feels a bit sledgehammery in this case.